serverutils.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	emailLib "github.com/jordan-wright/email"
+)
+
+func SendBulkEmail(emailAccounts []*EmailAccount, sendBulkEmailReq *SendBulkEmailRequest, emailPool *emailLib.Pool) []string {
```

Comment:
@ajvb If you set the return value to `(failedIds []string)` then you can remove line 11 altogether while making it clear what the `[]string` return value is/means/contains.

Reply:
fixed